prowler/providers/aws/services/lightsail/lightsail_service.py

```diff
@@ -28,7 +28,7 @@ def _get_instances(self, regional_client):
                         f"arn:{self.audited_partition}:lightsail:{regional_client.region}:{self.audited_account}:Instance",
                     )
 
-                    if not self.audit_resources or is_resource_filtered(
+                    if not self.audit_resources or (
```

Comment:
Could you please add some tests for this behaviour? We should verify that this behaviour is working as expected.